ruby-generate-dockerfile/app/Dockerfile.erb

```diff
@@ -49,24 +49,28 @@ RUN apt-get install -y -q <%= install_packages.join(' ') %>
 ## provided prebuilt binaries if available, and falls back to using rbenv.
 ## It sets the new version as the default, and also installs bundler.
 <% if ruby_version.empty? %>
-# ARG REQUESTED_RUBY_VERSION="2.4.2"
-# RUN if test ! -x /rbenv/versions/${REQUESTED_RUBY_VERSION}/bin/ruby; then \
-#       apt-get install -y -q gcp-ruby-${REQUESTED_RUBY_VERSION} \
-#       || (cd /rbenv/plugins/ruby-build \
+# ARG REQUESTED_RUBY_VERSION="2.4.3"
```

Comment:
Do you still need this section? It's commented out.

Reply:
Yeah, that's intentional; if the Dockerfile generator determines that this section is not needed, it still writes a commented-out version to the generated Dockerfile, inviting a takeout user to uncomment it if they need it.